conneg-by-ap/index.html

```diff
@@ -841,6 +841,43 @@ <h3>Alternate Representations Data Model</h3>
       </p>
     </section>
   </section>
+  <section id="link-attributes">
+    <h2>Link Attributes</h2>
+      <section id="link-attributes-token">
+	<h3>Token</h3>
+	<p>
+	  When used in a <code>Link</code> header field,
+	  the attribute <code>token</code> is used to specify
+	  the mapping between the profile URI
+	  (specified using the <code>anchor</code> attribute)
+	  and a <a>token</a> used as an alternative to the full profile URI.
+	</p>
+	<p>
+	  It should be noted that the scope of the mapping between the token and the profile URI
+	  is per default limited to the current resource
+	  (i. e. the resource identified by the current request URI).
+	  Servers MAY use a larger scope for their mappings
+	  but clients should not depend on that
+	  unless the server documentation explicitly gives other instructions.
+	</p>
+	<p>
+	  The ABNF for the profile attribute's value is <code>token</code>,
+	  where "token" is defined as in 
+		<a href="https://tools.ietf.org/html/rfc7230#section-3.2.6">section 3.2.6</a>
+	  of [[RFC7230]].
+	</p>
+	<pre id="eg-link-attribute-token" class="example nohighlight" aria-busy="false" aria-live="polite"
+           title="Using the Link attribute "token" to link a profile URI to a token">
+# The profile URI in the "anchor" element is linked to the token "px"
```

Comment:
Just wondering whether `px` is a realistic name of a token. I also reused it myself, but I have no idea if this would be typical. 2 chars seem really short.

Reply:
Yes, really short. I've been using things like *dcat* or *prov* (Turtle prefixes) or well-known things like ISO + standard numbers. See examples of tokens in old view/format system in use: http://pid.geoscience.gov.au/sample/AU100?_view=alternates

Reply:
^ let's perhaps use one of these?